src/sfizz/Region.cpp

```diff
@@ -579,13 +579,30 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode, bool cleanOpcode)
             filters[filterIndex].resonance = opcode.read(Default::filterResonance);
         }
         break;
-    case_any_ccN("cutoff&"): // also cutoff_oncc&, cutoff_cc&, cutoff&_cc&
+    case hash("cutoff&_cc&"): case hash("cutoff&_oncc&"):
         {
-            const auto filterIndex = opcode.parameters.front() - 1;
+            const auto filterIndex = opcode.parameters.empty() ? 0 : (opcode.parameters.front() - 1);
```

Comment:
the parameters shouldn't be empty here, otherwise the case won't match.

Reply:
Fair, though I'm pretty sure that conditional statement was there in the first place as error prevention against the index subtraction later on in the same line, even if it might have been excessive. Omit if needed.